dsgrid/config/dimensions.py

```diff
@@ -401,6 +406,7 @@ def check_frequency(cls, values):
 
     @validator("ranges", pre=True)
     def check_times(cls, ranges, values):
+        # TODO: Should this be a root_validator with pre=False, skip_on_failure=True?
```

Comment:
Delete pre and then check that values contains the required fields. Same for below.